symengine/tests/basic/test_parser.cpp

```diff
@@ -5,6 +5,7 @@
 #include <symengine/parser.h>
 #include <symengine/polys/basic_conversions.h>
 #include <symengine/symengine_exception.h>
+#include <symengine/parser/parser.h>
```

Comment:
This is an internal header and I don't think it's installed.

Reply:
It's installed here: https://github.com/symengine/symengine/pull/1591/files#diff-cdc0541896f88b9d0009ef8ad1007a3aR155. The parser has to be accessible, so I install all the headers in parser. Is there a downside to install all headers?

Reply:
> Is there a downside to install all headers?

These headers had macro names which could easily conflict with other headers. Is there a way to make `bison` and `re2c` prepend `SYMENGINE_` to all the macros?

Reply:
I can make the Bison header private, that's only used internally anyway.

Reply:
Looks like there aren't any macros in the new files. Can you move `struct YYSTYPE` inside symengine namespace?

Reply:
Fixed in 94ad5138237568d3a13376a313c61d92bea65e10.